src/agent/src/image_rpc.rs

```diff
@@ -450,26 +219,22 @@ impl ImageService {
                 &policy_path,
                 aa_kbc_params,
             )?;
+
+            Self::unpack_image(&cid)?;
         } else {
-            let image = image.to_string();
-            let cid = cid.to_string();
-            let source_creds =
-                (!req.get_source_creds().is_empty()).then(|| req.get_source_creds().to_string());
-            let aa_kbc_params = aa_kbc_params.to_string();
-
-            // ocicrypt-rs keyprovider module will create a new runtime to talk with
-            // attestation agent, to avoid startup a runtime within a runtime, we
-            // spawn a new thread here.
-            tokio::task::spawn_blocking(move || {
-                Self::pull_image_with_oci_distribution(&image, &cid, &source_creds, &aa_kbc_params)
-                    .map_err(|err| warn!(sl!(), "pull image failed: {:?}", err))
-                    .ok();
-            })
-            .await?;
+            let bundle_path = Path::new(CONTAINER_BASE).join(&cid);
+            fs::create_dir_all(&bundle_path)?;
+
+            info!(sl!(), "pull image {:?}, bundle path {:?}", cid, bundle_path);
+            // Image layers will store at IMAGE_RS_WORK_DIR, generated bundles
```

Comment:
> // Image layers will store at IMAGE_RS_WORK_DIR, generated bundles

I'm totally confused :confused: Do you mean `CC_IMAGE_WORK_DIR`? https://github.com/arronwy/image-rs uses `CC_IMAGE_WORK_DIR` (well sets it only?) But tbh I would have expected that crate to use variables with an `IMAGE_RS` prefix...?

Reply:
yes, the comment try to describe the work dir used by kata agent, maybe it will be less confusing after we defined as `KATA_CC_IMAGE_WORK_DIR`